When users chose to install the extension via the package manager, it is safe to
assume that they don't want to be bothered by popup messages when there is a new
upstream release of the extension available via other update channels.

Index: keepassxc-browser/keepassxc-browser/background/page.js
===================================================================
--- keepassxc-browser.orig/keepassxc-browser/background/page.js	2021-04-02 04:59:13.641558436 +0200
+++ keepassxc-browser/keepassxc-browser/background/page.js	2021-04-02 05:00:47.388318687 +0200
@@ -8,7 +8,7 @@
     autoReconnect: false,
     autoRetrieveCredentials: true,
     autoSubmit: false,
-    checkUpdateKeePassXC: 3,
+    checkUpdateKeePassXC: 0,
     clearCredentialsTimeout: 10,
     colorTheme: 'system',
     credentialSorting: SORT_BY_GROUP_AND_TITLE,
